HTTP_Engine/Compute/GetRequest.cs

```diff
@@ -52,19 +52,48 @@ public static string GetRequest(string uri)
 
         /***************************************************/
 
-        public static string GetRequest(string url, Dictionary<string, object> headers = null, Dictionary<string, object> parameters = null)
+        private static string GetRequest(string uri, Dictionary<string, object> headers = null)
         {
-            string uri = Convert.ToUrlString(url, parameters);
-            return GetRequest(uri);
+            HttpClient client = new HttpClient();
+
+            //Add headers
+            if (headers != null)
+            {
+                foreach (var kvp in headers)
+                {
+                    client.DefaultRequestHeaders.Add(kvp.Key, kvp.Value.ToString());
+                }
+            }
+
+            //Post login auth request and return token to m_bearerKey
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, uri);
+
+
+            using (HttpResponseMessage response = client.SendAsync(request).Result)
+            {
+                string result = response.Content.ReadAsStringAsync().Result;
+                if (!response.IsSuccessStatusCode)
+                {
+                    Engine.Reflection.Compute.ClearCurrentEvents();
+                    Engine.Reflection.Compute.RecordError($"GET request failed with code {response.StatusCode}: {response.ReasonPhrase}");
+                    return null;
+                }
+                else
+                {
+                    return result;
+                }
+            }
         }
 
         /***************************************************/
 
-        public static string GetRequest(string baseUrl, CustomObject headers = null, CustomObject parameters = null)
+        private static string GetRequest(string baseUrl, Dictionary<string, object> headers = null, Dictionary<string, object> parameters = null)
```

Comment:
Not sure I understand the changes. Why removing the `parameters` input? 
One of the differences between the two is that parameters can be part of the `uri`, e.g. `google.com/search?p1=v1,p2=v2` and so on.